src/main/java/com/minecolonies/rankup/util/ConfigUtils.java

```diff
@@ -0,0 +1,114 @@
+package com.minecolonies.rankup.util;
+
+import com.minecolonies.rankup.Rankup;
+import com.minecolonies.rankup.modules.core.CoreModule;
+import com.minecolonies.rankup.modules.core.config.AccountConfigData;
+import com.minecolonies.rankup.modules.core.config.CoreConfig;
+import com.minecolonies.rankup.modules.core.config.CoreConfigAdapter;
+import com.minecolonies.rankup.modules.core.config.GroupsConfig;
+import com.minecolonies.rankup.modules.databases.DatabaseModule;
+import com.minecolonies.rankup.modules.databases.config.DatabaseConfig;
+import com.minecolonies.rankup.modules.databases.config.DatabaseConfigAdapter;
+import com.minecolonies.rankup.modules.economy.EconomyModule;
+import com.minecolonies.rankup.modules.economy.config.EconomyConfig;
+import com.minecolonies.rankup.modules.economy.config.EconomyConfigAdapter;
+import com.minecolonies.rankup.modules.magibridge.MagibridgeModule;
+import com.minecolonies.rankup.modules.magibridge.config.MagibridgeConfig;
+import com.minecolonies.rankup.modules.magibridge.config.MagibridgeConfigAdapter;
+import com.minecolonies.rankup.modules.timing.TimingModule;
+import com.minecolonies.rankup.modules.timing.config.TimingConfig;
+import com.minecolonies.rankup.modules.timing.config.TimingConfigAdapter;
+import org.spongepowered.api.entity.living.player.Player;
+
+/**
+ * Tonnes of config Utils.. holy cow.
+ */
+public class ConfigUtils
+{
+
+    Rankup plugin;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Explicitly declare the visibility for "plugin". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2039)